index.html

```diff
@@ -356,10 +317,9 @@ <h2 class="title_contact">Contact me</h2>
 					</li>
 				</ul>
 			</form>
-			<div class="bar"></div>
+			<footer class="bar"></footer>
```

Comment:
- [ ] [OPTIONAL] As already pointed out, this `footer` tag should wrap the `form` and its elements